src/components/Footer/Footer.tsx

```diff
@@ -1,14 +1,58 @@
+import { HiChevronRight } from "react-icons/hi";
 import styles from "./Footer.module.scss";
 
 export interface FooterProps {
   linkGroups: { label: string; links: { href: string; label: string }[] }[];
 }
 
 export const Footer: React.FC<FooterProps> = ({ linkGroups }) => {
+  const year = new Date().getFullYear();
+
+  const content = linkGroups.map((group) => {
+    return (
+      <div key={group.label} className={styles.category}>
+        <h2 className={styles.categoryLabel}>{group.label}</h2>
+        <ul key={group.label} className={styles.categoryList}>
+          {group.links &&
```

Comment:
Don't need to check for `group.links` since you can expect it to be an array based on the prop types